ci_scripts/dashboard/EnvironmentModules.cmake

```diff
@@ -0,0 +1,207 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
this file would presumably be distributed with cmake

Reply:
This is not distributed with CMake, but you mentioned last week that you don't want to use this anyway, correct?  Along those lines, I just want to be clear what it is you do want:  

1. Do you want us to provide a shell script instead that invokes the `module load ...` command (leaving the environment module installed in the Docker image) so CTest can find the tests to run and the needed libraries?

2. Or did you want to get rid of the environment modules from the Docker image completely and just update the `PATH`, `PYTHONPATH`, and `LD_LIBRARY_PATH` manually before we run the tests?

With either approach above, we get rid of the `EnvironmentModules.cmake` file.